proxy/src/coco.rs

```diff
@@ -88,20 +90,32 @@ impl Peer {
         clippy::wildcard_enum_match_arm,
         clippy::match_wildcard_for_single_variants
     )]
-    pub fn list_projects(&self) -> Result<Vec<project::Project<entity::Draft>>, error::Error> {
-        self.with_api(|api| {
-            let storage = api.storage();
```

Comment:
We have the api lock here. I ran into double-locking using `with_browser` but maybe we can make the change cleaner. Not sure :shrug: